diff -urN fetchmail-6.2.5.org/socket.c fetchmail-6.2.5/socket.c
--- fetchmail-6.2.5.org/socket.c	2004-08-20 01:45:26.521602080 +0200
+++ fetchmail-6.2.5/socket.c	2004-08-20 01:46:14.535302888 +0200
@@ -1009,6 +1009,7 @@
 		
 		if( NULL != ( ssl = SSLGetContext( sock ) ) ) {
 			/* Clean up the SSL stack */
+		    	SSL_shutdown( _ssl_context[sock] );
 			SSL_free( _ssl_context[sock] );
 			_ssl_context[sock] = NULL;
 		}
@@ -1027,6 +1028,7 @@
 
     if( NULL != ( ssl = SSLGetContext( sock ) ) ) {
         /* Clean up the SSL stack */
+	SSL_shutdown( _ssl_context[sock] );
         SSL_free( _ssl_context[sock] );
         _ssl_context[sock] = NULL;
     }
